advection/conservative_sw_mhd.jl

```diff
@@ -26,13 +27,27 @@ model = ShallowWaterModel(grid = grid,
                           formulation = ConservativeFormulation()
                           )
 
-Aᵢ(x, y, z) = -0.05*y
+Aᵢ(x, y, z) = 0.1*exp(-((x - 0.5)^2 + y^2)) - 0.1*exp(-((x + 0.5)^2 + y^2))
 
 hᵢ(x, y, z) = 1
-uhᵢ(x, y, z) = y*exp(-(x^2 + y^2))
-vhᵢ(x, y, z) = -x*exp(-(x^2 + y^2))
-set!(model, uh = uhᵢ, vh = vhᵢ, h = hᵢ, A = Aᵢ)
-simulation = Simulation(model, Δt = 0.01, stop_time = 25)
+#uhᵢ(x, y, z) = y*exp(-(x^2 + y^2))
+#vhᵢ(x, y, z) = -x*exp(-(x^2 + y^2))
+set!(model,#= uh = uhᵢ, vh = vhᵢ,=# h = hᵢ, A = Aᵢ)
+simulation = Simulation(model, Δt = 0.01, stop_time = 70)
+
+uh, vh, h = simulation.model.solution
+u = uh / h
+v = vh / h
+A = simulation.model.tracers.A
+B_x = -∂y(A) / h
+B_y = ∂x(A) / h
+kinetic_term = h*(u^2 + v^2)
+magnetic_term = h*(B_x^2 + B_y^2)
+pressure_term = 9.81*h^2
+energy_term = compute!(kinetic_term + magnetic_term + pressure_term)
```

Comment:
term is not very specific.  How about we use the following instead?
kinetic_energy
magnetic_energy
potential_energy

Reply:
I was thinking since each of these terms correspond to their corresponding types of energy that it'd be sufficient to say kinetic term etc (but I agree your suggestions would be more specific)

Reply:
If we have to have a word, I prefer energy, as its more specific than term, which could be anything.